Simulations/runESOsmall.sh

```diff
@@ -1,13 +1,12 @@
-#!/bin/bash
 
-cat YAML/persist.yaml > YAML/allRecipes.yaml
-cat YAML/imgLM.yaml >> YAML/allRecipes.yaml
-cat YAML/imgN.yaml >> YAML/allRecipes.yaml
-cat YAML/lssLM.yaml >> YAML/allRecipes.yaml
-cat YAML/lssN.yaml >> YAML/allRecipes.yaml
-cat YAML/ifu.yaml >> YAML/allRecipes.yaml
-cat YAML/hci.yaml >> YAML/allRecipes.yaml
-cat YAML/calib.yaml >> YAML/allRecipes.yaml
-
-python/run_recipes.py --inputYAML=YAML/allRecipes.yaml --outputDir outputSmall/ --doCalib=1 --sequence=1 --doStatic --nCores=8 --small
+# run the simulation blocks for all instruments
 
+python/imgLM_small.py
+python/imgN_small.py
+python/lssLM_small.py
+python/lssN_small.py
+python/ifu_small.py
+python/calib_small.py
+python/hciRavcLM_small.py
+python/hciAppLM_small.py
+python/hciRavcIFU_small.py
```

Comment:
You seem to have forgotten the `python` executable part:
```suggestion
python python/imgLM_small.py
python python python/imgN_small.py
python python python/lssLM_small.py
python python python/lssN_small.py
python python/ifu_small.py
python python/calib_small.py
python python/hciRavcLM_small.py
python python/hciAppLM_small.py
python python/hciRavcIFU_small.py
```
But I think making separate `*_small.py` files kinda defeats the point.

But those small files have almost served their purpose though, so maybe we should remove the functionality soon anyway, so it doesn't matter now.

Reply:
We were discussing this yesterday, and the small files are basically obsolete now for pipeline work. I'll keep the small option for the github runner, so we can run tests without using a ton of space.